show/main.py

```diff
@@ -2806,5 +2806,44 @@ def features():
         body.append([key, status_data[key]['status']])
     click.echo(tabulate(body, header))
 
+#
+# 'container' group (show container ...)
+#
+@cli.group(name='container', invoke_without_command=False)
+def container()
+    """Show container"""
+    pass
+#
+# 'feature' group (show container feature ...)
+#
+@container.group(name='feature', invoke_without_command=False)
+def feature()
+    """Show container feature"""
+    pass
+
+#
+# 'autorestart' subcommand (show container feature autorestart)
+#
+
+@feature.command('autorestart')
+@click.argument('container_name', required=False)
+def autorestart(container_name):
+    """show whether the auto-restart feature for containers or a specific container
+       is enabled or disabled.
```

Comment:
Good, but a bit too wordy. Try, `Show whether the auto-restart feature for container(s) is enabled or disabled`

Also, if the docstring is longer than 45 characters, it will get truncated. However, I discovered recently that one can work around this by defining the parameter `short_help` instead of using a docstring (see https://github.com/pallets/click/issues/486). I think we should probably just move over to this method everywhere. Can you please try that method instead? Example:

```
@feature.command('autorestart', short_help="Show whether the auto-restart feature for container(s) is enabled or disabled")
```

Reply:
Will try the second method.

Reply:
I tested the second method on DUT and It did work!